pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, lib
+, openssl
+, pkg-config
+, protobuf
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "prisma-engines";
+  version = "2.30.0";
+
+  src = fetchFromGitHub {
+    owner = "prisma";
+    repo = "prisma-engines";
+    rev = version;
+    sha256 = "sha256-i4r+TRC8454awbqe35Kg3M9xN2NnP8Sbd/dITtm9MDg=";
+  };
+
+  cargoPatches = [
+    # Remove test from compilation targets:
+    # they add time to an already long compilation and some fail out-of-the-box.
```

Comment:
I'm not saying you should, but if you want, cargo provides ways to skip particular tests.
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/vector/default.nix#L63
You can add it as cargoCheckFlags
(Personally this is good to merge as is. Just mentioning it in case you are interested)

Reply:
Would that skip their compilation too, or just their run?
Because feels like adding tests to an already couple of minutes long compilation, to skip them is kinda a waste...

Reply:
I don't know actually.

Reply:
I'd say just disabling the tests is a good thing to do for this build. They are HEAVY. The query engine holds about 15000 tests and now they're all generated on compilation.

We do have an extensive CI setup so I'd say running them again on Nix is probably not needed.